### DISCUSSION TITLE
Unable to start emqx  in Windows 10 getting escript.exe - System Error because MSVCR120.dll was not found

### DISCUSSION THREAD

**Discussion by vgkkonda** [emqx/emqx#5855] [Q&A]
Hi There,
I have downloaded the Open Source, 4.3.8 version for Windows from i.e. https://www.emqx.com/en/downloads/broker/4.3.8/emqx-windows-4.3.8.zip
And decompressed it and ran this command bin/emqx start

Getting following error in my Windows 10 machine.

**escript.exe - System Error
The code execution cannot proceed because MSVCR120.dll was not found. Reinstalling the probram may fix this problem.**

Can you please let me know how to fix this issue?

Thanks & Regards,
-Venkat

**Comment by k32** [comment]
Hello,
You probably need to install [Microsoft Visual C++ redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) package from the official Microsoft website.
